### ISSUE TITLE
Issue in ? server.status (0.6)

### ISSUE THREAD

**Issue by menthalo37** [rkrug/plantuml#34]
Hello, 
I found an issue with installation of 0.6 (current on 2022-10-12) version, in R (see sessionInfo below).
It seems to come from **server.status()** function, where response variable isn't properly managed (perhaps NULL value ?)

I precise installation of 0.4 version is OK on same computer/same R version, through an manual installation of (plantuml 0.4).zip file and using the following R code

Thanks for correct 0.6 version or precise how installation within R could be done.

```
fic <- "(directory where this file is stored)/plantuml-0.4.zip"
devtools::install_local(fic)
```

**Details on my R sessionInfo**

_R version 4.1.1 (2021-08-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19044))_

```
Error: le chargement du package ou de l'espace de noms a �chou� pour 'plantuml' :
.onAttach a �chou� dans attachNamespace() pour 'plantuml', d�tails :
  appel : if ((response != -404) & (response != -999)) {
  erreur : l'argument est de longueur nulle
Erreur : loading failed
Ex�cution arr�t�e
ERROR: loading failed
* removing ' (... my personal directory here ) plantuml'

Exited with status 1.
```

**Comment by menthalo37** [comment]
OK, no emergency for me (as far as 0.4 version works well). I wasn't able
to discover your site https://rkrug.r-universe.dev/ui#package:plantuml
unless you gave me the link, so I managed 0.4 installation through a manual
install (zip download fro github , then install from local got zip file).
tar.gz is easier from my point of view. Perhaps a good idea could be to put
your link directly in the github project description.
Best regards and have a good evening,
E.

Le mer. 12 oct. 2022 à 14:43, Rainer M Krug ***@***.***> a
écrit :

**Comment by menthalo37** [comment]
I meet errors again with use of your "local old version" 0.4 on this link : 
https://rkrug.r-universe.dev/ui#package:plantuml 


**> devtools::install(file.path(d,"plantuml"), dependencies = TRUE)**
```
These packages have more recent versions available.
It is recommended to update all of them.
Which would you like to update?

1: All                               
2: CRAN packages only                
3: None                              
4: vctrs    (0.4.2  -> 0.5.0 ) [CRAN]
5: ps       (1.7.1  -> 1.7.2 ) [CRAN]
6: processx (3.7.0  -> 3.8.0 ) [CRAN]
7: xfun     (0.33   -> 0.34  ) [CRAN]
8: stringi  (1.7.6  -> 1.7.8 ) [CRAN]
9: digest   (0.6.29 -> 0.6.30) [CRAN]

Enter one or more numbers, or an empty line to skip updates: 3
v  checking for file 'C:\Users\d37050\AppData\Local\Temp\RtmpS4cVGP\plantuml/DESCRIPTION' (2.3s)
-  preparing 'plantuml': (486ms)
v  checking DESCRIPTION meta-information ... 
   Avis dans file(con, "r") :
     impossible d'ouvrir le fichier 'man' : No such file or directory
    ERROR
   computing Rd index failed:impossible d'ouvrir la connexion
Error in `(function (command = NULL, args = character(), error_on_status = TRUE, ...`:
! System command 'Rcmd.exe' failed
---
Exit status: 1
stdout & stderr: <printed>
---
Type .Last.error to see the more details.
```

**Comment by rkrug** [comment]
Thanks a lot. Should be fixed in 0.6.5

Sorry about the horrible release v.0.6.0
